Transfer/Sources/CreateTransferController.swift

```diff
@@ -234,11 +234,9 @@ extension CreateTransferController {
         if let transferMethod = presenter.selectedTransferDestination {
             tableViewCell.configure(transferMethod: transferMethod)
         } else {
-            if selectTransferMethodCoordinator == nil {
-                tableViewCell.accessoryType = .none
-            }
+            tableViewCell.accessoryType = .none
```

Comment:
Why is this changed?

Reply:
To hide the arrow mark next to Add Transfer Method title.